### ISSUE TITLE
make error: ‘glob’ was not declared in this scope

### ISSUE THREAD

**Issue by xzhub** [AlexeyAB/Yolo_mark#55]
I am trying to compile it under centos 7.

I have to change the file path of highgui.hpp to find the file in line 16 of main.cpp:

` 16 #include <opencv2/highgui/highgui.hpp>
`

but still, I got this error. I tried to check all the opencv2 .h and .hpp file, seems there is no such glob function. 

```
$ make
[100%] Building CXX object CMakeFiles/yolo_mark.dir/main.cpp.o
/home/xuzh/yolo/Yolo_mark/main.cpp: In function ‘int main(int, char**)’:
/home/xuzh/yolo/Yolo_mark/main.cpp:170:46: error: ‘glob’ was not declared in this scope
   glob(images_path_cv, filenames_in_folder_cv); // void glob(String pattern, std::vector<String>& result, bool recursive = false);
                                              ^
make[2]: *** [CMakeFiles/yolo_mark.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/yolo_mark.dir/all] Error 2
make: *** [all] Error 2
```

Packages I've installed:
```
$ rpm -qa |grep opencv
opencv-core-2.4.5-3.el7.x86_64
opencv-python-2.4.5-3.el7.x86_64
opencv-2.4.5-3.el7.x86_64
opencv-devel-2.4.5-3.el7.x86_64
```

Please help!

**Comment by AlexeyAB** [comment]
Hi, your version of opencv-core-2.4.5 is very old ~2014 year.

Try to install OpenCV 2.4.13 or OpenCV 3.4.0.

There is function `glob()`: https://github.com/opencv/opencv/blob/0d6518aaa05bc66b5724844938b6920627c5f13c/modules/core/include/opencv2/core/utility.hpp#L186

**Comment by xzhub** [comment]
Thanks ! Issue resolved by installing OpenCV 2.4.13

```
sudo yum erase opencv-core opencv-python opencv opencv-devel
wget https://github.com/opencv/opencv/archive/2.4.13.6.zip
unzip 2.4.13.6.zip
cd opencv-2.4.13.6/
mkdir build_release
cd build_release/
cmake -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/ ..
make
sudo make install
```
